### PR TITLE
Use dartcli from install tab

### PR DESCRIPTION
**Pull request by sigurdm**
Explain how to use `dart pub add`.

install

### CODE REVIEW

File: app/lib/frontend/templates/views/pkg/install_tab.mustache [hunk 1]
```diff
@@ -16,28 +16,26 @@
...
-</code></pre>
+<p>With Dart:</p>
+<pre><code class="language-shell"> $ dart pub add {{package}}</code></pre>
 {{/use_pub_get}}
 {{#use_flutter_packages_get}}
-  <p>with Flutter:</p>
-  <pre><code class="language-shell">$ <strong>flutter pub get</strong>
-</code></pre>
+<p>With Flutter:</p>
+<pre><code class="language-shell"> $ flutter pub pub add {{package}}</code></pre>
```
Comment (jonasfj) [inline app/lib/frontend/templates/views/pkg/install_tab.mustache:26]: Maybe we shouldn't show this part

Comment (sigurdm) [reply app/lib/frontend/templates/views/pkg/install_tab.mustache:26]: Yeah - maybe this is too early. We need `flutter pub add` to work first...

**Comment by isoos** [comment]
`dart pub add` returns `Could not find a command named "add".` on the stable SDK. Maybe we should wait with this a bit?

**Comment by isoos** [comment]
rebased / updated in #4687

**Pull request closed without merging**